perf/perftool.py

```diff
@@ -39,7 +39,15 @@ def setUp(self):
 
         # Check for basic utilities
         smm = SoftwareManager()
-        for package in ['gcc', 'make', 'perf']:
+        detected_distro = distro.detect()
+        kernel_ver = platform.uname()[2]
+        if 'Ubuntu' in detected_distro.name:
+            deps = ['gcc', 'make', 'linux-tools-common',
+                    'linux-tools-' + kernel_ver + '']
+        else:
+            deps = ['gcc', 'make', 'perf']
+
+        for package in deps:
             if not smm.check_installed(package) and not smm.install(package):
                 self.error('%s is needed for the test to be run' % package)
```

Comment:
one request as you submitting changes can you please change  `self.error` to` self.skip`